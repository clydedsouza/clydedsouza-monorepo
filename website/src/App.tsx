import Mainstage from './Components/Mainstage/Mainstage'
import Sidebar from './Components/Sidebar/Sidebar'

export function App() {
  return (
    <>
      <Sidebar />
      <Mainstage />
    </>
  )
}
