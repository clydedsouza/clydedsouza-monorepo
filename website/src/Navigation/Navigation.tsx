import { Outlet, Link } from "react-router-dom";

function Navigation() {
    return (
      <div>
       <nav>
        <ul>
          <li>
            <Link to="/">Home</Link>
          </li>
          <li>
            <Link to="/projects">Blogs</Link>
          </li>
          <li>
            <Link to="/headline">Healdine</Link>
          </li>
        </ul>
      </nav>

      <Outlet />
      </div>
    );
  }
  
  export default Navigation;