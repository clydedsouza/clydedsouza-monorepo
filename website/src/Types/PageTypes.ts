export enum PageTypes {
  About = 'About',
  Projects = 'Projects',
  Speaking = 'Speaking',
  Teaching = 'Teaching',
  Books = 'Books',
  Highlights = 'Highlights',
  Pinned = 'Pinned',
}
