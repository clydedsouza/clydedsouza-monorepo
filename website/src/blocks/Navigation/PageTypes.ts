export enum PageTypes {
  About = "About",
  Portfolio = "Portfolio",
  Platforms = "Platforms",
  Highlights = "Highlights",
  Cta = "Cta",
  Blog = "Blog",
}
