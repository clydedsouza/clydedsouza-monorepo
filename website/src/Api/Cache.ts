import { PageTypes } from '../Types/PageTypes'
import { IProjectData } from '../Types/ProjectData'
import { getProjectData } from './Core'

const projectDataCacheMap: Map<PageTypes, Promise<IProjectData>> = new Map()

export const getCachedProjectData = (
  pageType: PageTypes
): Promise<IProjectData> => {
  if (!projectDataCacheMap.has(pageType)) {
    const getProjectDataPromise = getProjectData(pageType).catch((error) => {
      projectDataCacheMap.delete(pageType)
      return Promise.reject(error)
    })
    projectDataCacheMap.set(pageType, getProjectDataPromise)
  }

  return projectDataCacheMap.get(pageType)!
}
