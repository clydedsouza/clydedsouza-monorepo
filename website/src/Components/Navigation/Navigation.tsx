import { BrowserRouter, NavLink, Outlet, Route, Routes } from 'react-router-dom'
import AboutMe from '../../Pages/AboutMe/AboutMe'
import Projects from '../../Pages/Projects/Projects'
import { PageTypes } from '../../Types/PageTypes'
import './Navigation.scss'

function Menu() {
  return (
    <>
      <nav>
        <NavLink
          to="/"
          className={({ isActive }) => (isActive ? 'active' : '')}
        >
          {PageTypes.About}
        </NavLink>
        <NavLink
          to={PageTypes.Highlights.toLowerCase()}
          className={({ isActive }) => (isActive ? 'active' : '')}
        >
          {PageTypes.Highlights}
        </NavLink>
        <NavLink
          to={PageTypes.Platforms.toLowerCase()}
          className={({ isActive }) => (isActive ? 'active' : '')}
        >
          {PageTypes.Platforms}
        </NavLink>
      </nav>
      <Outlet />
    </>
  )
}

function Navigation() {
  return (
    <BrowserRouter>
      <Routes>
        <Route path="/" element={<Menu />}>
          <Route index element={<AboutMe />} />
          <Route
            path={PageTypes.Highlights.toLowerCase()}
            element={<Projects name={PageTypes.Highlights} />}
          />
          <Route
            path={PageTypes.Platforms.toLowerCase()}
            element={<Projects name={PageTypes.Platforms} />}
          />
          <Route path="*" element={<AboutMe />} />
        </Route>
      </Routes>
    </BrowserRouter>
  )
}

export default Navigation
