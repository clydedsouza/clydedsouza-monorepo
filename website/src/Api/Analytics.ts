import mixpanel from 'mixpanel-browser'
import { PageTypes } from '../Components/Navigation/PageTypes'
import {
  ILinkClickedProps,
  MIXPANEL_DEV_PROJECT_ID,
  MIXPANEL_PROD_PROJECT_ID,
} from './IAnalytics'

export const initAnalyticsWithSuperProperties = () => {
  const mixpanelProjectId =
    process.env.NODE_ENV === 'production'
      ? MIXPANEL_PROD_PROJECT_ID
      : MIXPANEL_DEV_PROJECT_ID

  try {
    mixpanel.init(mixpanelProjectId)
    mixpanel.register({
      environment: process.env.NODE_ENV,
    })
  } catch {
    undefined
  }
}

export const sendPageViewEvent = (pageType: PageTypes) => {
  try {
    mixpanel.track('Page view', { page: pageType })
  } catch {
    undefined
  }
}

export const sendLinkClickedEvent = (props: ILinkClickedProps) => {
  try {
    mixpanel.track('Link clicked', {
      destination_url: props.link,
      link_type: props.type,
      page_location: props.location,
    })
  } catch {
    undefined
  }
}
