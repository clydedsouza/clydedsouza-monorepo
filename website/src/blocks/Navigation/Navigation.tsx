"use client";

import { sendNavigationClickedEvent } from "@/api/Analytics";
import Link from "next/link";
import { usePathname } from "next/navigation";
import { useState } from "react";
import "./Navigation.scss";
import { PageTypes } from "./PageTypes";

const getCurrentPage = (pathFromNextJs: string) => {
  const cleanPath = pathFromNextJs.replaceAll("/", "").toLowerCase();

  if (cleanPath === PageTypes.Platforms.toLowerCase()) {
    return PageTypes.Platforms;
  } else if (cleanPath === PageTypes.Portfolio.toLowerCase()) {
    return PageTypes.Portfolio;
  } else {
    return PageTypes.About;
  }
};

function Navigation() {
  const [currentPage, setCurrentPage] = useState(getCurrentPage(usePathname()));

  const onNavigationItemClick = (pageType: PageTypes) => {
    sendNavigationClickedEvent(pageType);
    setCurrentPage(pageType);
  };

  return (
    <>
      <nav>
        <Link
          href="/"
          className={currentPage === PageTypes.About ? "active" : ""}
          onClick={() => onNavigationItemClick(PageTypes.About)}
        >
          {PageTypes.About}
        </Link>
        <Link
          href={PageTypes.Portfolio.toLowerCase()}
          className={currentPage === PageTypes.Portfolio ? "active" : ""}
          onClick={() => onNavigationItemClick(PageTypes.Portfolio)}
        >
          {PageTypes.Portfolio}
        </Link>
        <Link
          href={PageTypes.Platforms.toLowerCase()}
          className={currentPage === PageTypes.Platforms ? "active" : ""}
          onClick={() => onNavigationItemClick(PageTypes.Platforms)}
        >
          {PageTypes.Platforms}
        </Link>
        <Link
          href={"https://blog.clydedsouza.net"}
          target="_blank"
          onClick={() => sendNavigationClickedEvent(PageTypes.Blog)}
        >
          {PageTypes.Blog}
        </Link>
      </nav>
    </>
  );
}

export default Navigation;
