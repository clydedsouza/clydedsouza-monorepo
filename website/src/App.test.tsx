import {
  render,
  screen,
  waitForElementToBeRemoved,
} from '@testing-library/react'
import { emptyProject } from '../TestHelpers/testHelper'
import * as apiCacheModule from './Api/Cache'
import { App } from './App'
import { PageTypes } from './Components/Navigation/PageTypes'
jest.mock('./Api/Cache')

const apiDataMockResponse = [
  {
    ...emptyProject,
    id: 'ai-tell-me-a-story',
    title: 'Purchase my book AI, Tell Me a Story from these platforms.',
    website: 'https://aitellmeastory.clydedsouza.net',
  },
]

describe('App', () => {
  beforeEach(() => {
    jest.spyOn(apiCacheModule, 'getCachedProjectData').mockImplementation(() =>
      Promise.resolve({
        app: {},
        data: [apiDataMockResponse[0]],
      })
    )
  })

  it('should render app', async () => {
    render(<App />)
    await waitForElementToBeRemoved(() => screen.queryAllByRole('progressbar'))
    expect(document.body).toMatchSnapshot()
  })

  it('should call the API endpoint thrice', async () => {
    render(<App />)
    await waitForElementToBeRemoved(() => screen.queryAllByRole('progressbar'))
    expect(apiCacheModule.getCachedProjectData).toBeCalledTimes(3)
    expect(apiCacheModule.getCachedProjectData).toBeCalledWith(PageTypes.Cta)
    expect(apiCacheModule.getCachedProjectData).toBeCalledWith(PageTypes.Cta)
    expect(apiCacheModule.getCachedProjectData).toBeCalledWith(
      PageTypes.Highlights
    )
  })
})
