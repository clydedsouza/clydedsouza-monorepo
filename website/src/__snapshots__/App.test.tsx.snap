// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render app 1`] = `
<body>
  <div>
    <div
      class="sidebar"
    >
      <div
        class="avatar"
      />
      <div
        class="headline"
      >
        <h1>
          Clyde D'Souza
        </h1>
        <div>
          <p>
            Software Engineer and Author
          </p>
          <p>
            <small>
              Auckland, New Zealand
            </small>
          </p>
        </div>
      </div>
      <div
        class="social-icons"
      >
        <span>
          <a
            href="https://linkedin.com/in/clydedz/"
            rel="noreferrer"
            target="_blank"
          >
            <svg
              class="linkedin-svg"
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
              />
            </svg>
          </a>
        </span>
        <span>
          <a
            href="https://twitter.com/clydedz"
            rel="noreferrer"
            target="_blank"
          >
            <svg
              class="twitter-svg"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M389.2 48h70.6L305.6 224.2 487 464H345L233.7 318.6 106.5 464H35.8L200.7 275.5 26.8 48H172.4L272.9 180.9 389.2 48zM364.4 421.8h39.1L151.1 88h-42L364.4 421.8z"
              />
            </svg>
          </a>
        </span>
        <span>
          <a
            href="https://www.youtube.com/@clydedz"
            rel="noreferrer"
            target="_blank"
          >
            <svg
              class="youtube-svg"
              viewBox="0 0 576 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M549.655 124.083c-6.281-23.65-24.787-42.276-48.284-48.597C458.781 64 288 64 288 64S117.22 64 74.629 75.486c-23.497 6.322-42.003 24.947-48.284 48.597-11.412 42.867-11.412 132.305-11.412 132.305s0 89.438 11.412 132.305c6.281 23.65 24.787 41.5 48.284 47.821C117.22 448 288 448 288 448s170.78 0 213.371-11.486c23.497-6.321 42.003-24.171 48.284-47.821 11.412-42.867 11.412-132.305 11.412-132.305s0-89.438-11.412-132.305zm-317.51 213.508V175.185l142.739 81.205-142.739 81.201z"
              />
            </svg>
          </a>
        </span>
        <span>
          <a
            href="https://github.com/ClydeDz/"
            rel="noreferrer"
            target="_blank"
          >
            <svg
              class="github-svg"
              viewBox="0 0 496 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
              />
            </svg>
          </a>
        </span>
        <span>
          <a
            href="https://medium.com/@clydedz"
            rel="noreferrer"
            target="_blank"
          >
            <svg
              class="medium-svg"
              viewBox="0 0 640 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M180.5,74.262C80.813,74.262,0,155.633,0,256S80.819,437.738,180.5,437.738,361,356.373,361,256,280.191,74.262,180.5,74.262Zm288.25,10.646c-49.845,0-90.245,76.619-90.245,171.095s40.406,171.1,90.251,171.1,90.251-76.619,90.251-171.1H559C559,161.5,518.6,84.908,468.752,84.908Zm139.506,17.821c-17.526,0-31.735,68.628-31.735,153.274s14.2,153.274,31.735,153.274S640,340.631,640,256C640,171.351,625.785,102.729,608.258,102.729Z"
              />
            </svg>
          </a>
        </span>
        <span>
          <a
            href="mailto:clyde@clydedsouza.net?Subject=Hi%20Clyde%21&Body=Hi%20Clyde,%20%0D%0DI%20wanted%20to%20get%20in%20touch%20with%20you.%0D%0DThanks,%20%0D%0DSent%20from%20https%3A%2F%2Fclydedsouza.net"
            rel="noreferrer"
            target="_blank"
          >
            <svg
              class="envelope-svg"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M48 64C21.5 64 0 85.5 0 112c0 15.1 7.1 29.3 19.2 38.4L236.8 313.6c11.4 8.5 27 8.5 38.4 0L492.8 150.4c12.1-9.1 19.2-23.3 19.2-38.4c0-26.5-21.5-48-48-48H48zM0 176V384c0 35.3 28.7 64 64 64H448c35.3 0 64-28.7 64-64V176L294.4 339.2c-22.8 17.1-54 17.1-76.8 0L0 176z"
              />
            </svg>
          </a>
        </span>
      </div>
      <div
        class="cta"
        data-location="sidebar"
      >
        <a
          href="https://aitellmeastory.clydedsouza.net"
          rel="noreferrer"
          target="_blank"
        >
          Purchase my book AI, Tell Me a Story from these platforms.
        </a>
      </div>
    </div>
    <main>
      <nav>
        <a
          aria-current="page"
          class="active"
          href="#/"
        >
          About
        </a>
        <a
          class=""
          href="#/portfolio"
        >
          Portfolio
        </a>
        <a
          class=""
          href="#/platforms"
        >
          Platforms
        </a>
      </nav>
      <h2>
        About me
      </h2>
      <p>
        Hi! I'm Clyde D'Souza. I'm a creative and self-driven person who likes to code and do other things. I'm currently working as a senior software engineer at
         
        <a
          href="https://www.xero.com/nz/?ref=clydedsouza.net"
          rel="noreferrer"
          target="_blank"
        >
          Xero
        </a>
        . Previously, I've worked with
         
        <a
          href="https://www.datacom.co.nz/?ref=clydedsouza.net"
          rel="noreferrer"
          target="_blank"
        >
          Datacom
        </a>
         
        and
         
        <a
          href="https://www.heritagehotels.co.nz/?ref=clydedsouza.net"
          rel="noreferrer"
          target="_blank"
        >
          Heritage Hotels
        </a>
        .
      </p>
      <p>
        I was the event director of a non-profit educational event called Light & Spark NPO. I conceptualized and executed this event in Mumbai along with my
         
        <a
          href="https://lightandsparknpo.github.io/team/"
          rel="noreferrer"
          target="_blank"
        >
          amazing team
        </a>
        . Read more about it
         
        <a
          href="https://lightandsparknpo.github.io/2019-11-20-ruia-mumbai-2018/"
          rel="noreferrer"
          target="_blank"
        >
          here
        </a>
        .
      </p>
      <p>
        I've published a book titled Mama, Tell Me a Story which is available on
         
        <a
          href="http://mamatellmeastory.clydedsouza.net/"
          rel="noreferrer"
          target="_blank"
        >
          these platforms
        </a>
        . It's a collection of twelve short bedtime stories that parents will love reading to their kids over and over again. Go buy it now!
      </p>
      <p>
        In my spare time, I like to explore other interests like
         
        <a
          href="https://medium.com/@clydedz"
          rel="noreferrer"
          target="_blank"
        >
          writing
        </a>
        ,
         
        <a
          href="https://www.behance.net/clydedz"
          rel="noreferrer"
          target="_blank"
        >
          designing
        </a>
        ,
         
        <a
          href="https://www.skillshare.com/r/user/clydedsouza"
          rel="noreferrer"
          target="_blank"
        >
          teaching
        </a>
        , and I've also created many other projects.
      </p>
      <p>
        I'm an alumnus of
         
        <a
          href="https://www.aut.ac.nz/?ref=clydedsouza"
          rel="noreferrer"
          target="_blank"
        >
          Auckland University of Technology (AUT)
        </a>
         
        with a Postgraduate Diploma in Computer and Information Sciences and holds a Bachelor of Science degree specializing in Information Technology from University of Mumbai, India.
      </p>
      <br />
      <h2>
        Highlights
      </h2>
      <div
        class="cards"
      >
        <div
          class="card"
        >
          <div
            aria-label=""
          />
          <div>
            <h3>
              Purchase my book AI, Tell Me a Story from these platforms.
            </h3>
            <p />
            <div
              class="card-links"
            >
              <a
                href="https://aitellmeastory.clydedsouza.net"
                rel="noreferrer"
                target="_blank"
              >
                <svg
                  class="globe-svg"
                  viewBox="0 0 512 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M352 256c0 22.2-1.2 43.6-3.3 64H163.3c-2.2-20.4-3.3-41.8-3.3-64s1.2-43.6 3.3-64H348.7c2.2 20.4 3.3 41.8 3.3 64zm28.8-64H503.9c5.3 20.5 8.1 41.9 8.1 64s-2.8 43.5-8.1 64H380.8c2.1-20.6 3.2-42 3.2-64s-1.1-43.4-3.2-64zm112.6-32H376.7c-10-63.9-29.8-117.4-55.3-151.6c78.3 20.7 142 77.5 171.9 151.6zm-149.1 0H167.7c6.1-36.4 15.5-68.6 27-94.7c10.5-23.6 22.2-40.7 33.5-51.5C239.4 3.2 248.7 0 256 0s16.6 3.2 27.8 13.8c11.3 10.8 23 27.9 33.5 51.5c11.6 26 20.9 58.2 27 94.7zm-209 0H18.6C48.6 85.9 112.2 29.1 190.6 8.4C165.1 42.6 145.3 96.1 135.3 160zM8.1 192H131.2c-2.1 20.6-3.2 42-3.2 64s1.1 43.4 3.2 64H8.1C2.8 299.5 0 278.1 0 256s2.8-43.5 8.1-64zM194.7 446.6c-11.6-26-20.9-58.2-27-94.6H344.3c-6.1 36.4-15.5 68.6-27 94.6c-10.5 23.6-22.2 40.7-33.5 51.5C272.6 508.8 263.3 512 256 512s-16.6-3.2-27.8-13.8c-11.3-10.8-23-27.9-33.5-51.5zM135.3 352c10 63.9 29.8 117.4 55.3 151.6C112.2 482.9 48.6 426.1 18.6 352H135.3zm358.1 0c-30 74.1-93.6 130.9-171.9 151.6c25.5-34.2 45.2-87.7 55.3-151.6H493.4z"
                  />
                </svg>
              </a>
              <a
                href="https://github.com"
                rel="noreferrer"
                target="_blank"
              >
                <svg
                  class="github-svg"
                  viewBox="0 0 496 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                  />
                </svg>
              </a>
            </div>
          </div>
        </div>
      </div>
      <footer>
        <div
          class="cta"
          data-location="footer"
        >
          <a
            href="https://aitellmeastory.clydedsouza.net"
            rel="noreferrer"
            target="_blank"
          >
            Purchase my book AI, Tell Me a Story from these platforms.
          </a>
        </div>
        <p>
          Website crafted by Clyde DSouza
        </p>
      </footer>
    </main>
  </div>
</body>
`;
